tfhe/src/high_level_api/config.rs

```diff
@@ -53,13 +65,18 @@ impl ConfigBuilder {
     pub fn with_custom_parameters<P>(
         block_parameters: P,
         wopbs_block_parameters: Option<crate::shortint::WopbsParameters>,
+        compression_parameters: Option<CompressionParameters>,
```

Comment:
Is it possible to not require the `compression_parameters` in this constructor ?

I prefer the API
```
Builder::with_custom_parameters(params, None)
   .enable_glwe_packing_compression(compression_params)
   .build()

```

rather than 
```
Builder::with_custom_parameters(params, None, compression_params)
   .build()
```

In hindsight this is probably how I should have made it for Wopbs

And so also avoid making a breaking change to the with_parameters constructor (not that it matters that much)

Reply:
Maybe we could keep things as they are and open a issue to refactor both later, wdyt?
It keeps the API consistent between WoP and compression params

Reply:
I made the change for my PR it's not a big change and it makes managing conflicts easier

Reply:
Keeping things consistent with WOP should not be a goal as its integration is a bit Frankenstein-Tier 😂

Reply:
yeah, it's getting the boot as well in the next quarter as it will be moved as an experimental feature, so we'll remove some Franken stuff from it